semgrep/semgrep/formatter/text.py

```diff
@@ -237,10 +238,10 @@ def _build_text_output(
             )
 
             if fix:
-                yield f"{with_color('bright_blue', 'autofix:')} {fix}"
+                yield f"{with_color('green', '     ▶▶ Autofix')} {fix}"
```

Comment:
@adamboros What do you think about having autofix _after_ the issue? I was always confused by seeing it show up before the line

Reply:
@ievans is there any case the autofix shows something different for each snippet on the same finding? Cause might be useful to get rid of the repetition as well if not.

Reply:
I think it can be different, because it can use metavars